var/spack/repos/builtin/packages/slepc/test/hello.c

```diff
@@ -0,0 +1,14 @@
+static char help[] = "Hello World example program\n";
```

Comment:
Is this required?

Reply:
I need it as an argument for `SlepcInitialize` so that I can use SLEPc classes.